### ISSUE TITLE
Bug — Schedule: Visual Padding in Course Elements

### ISSUE THREAD

**Issue by jshom** [YACS-RCOS/yacs.n#209]
**Describe the bug**
Visual padding issue in schedule view with course elements.
Course listing elements in the search are taking too much vertical height and the padding appears inconsistent.

**To Reproduce**
1. load semester
2. click on a course in schedule view

**Expected behavior**
Consistent padding around text content

**Screenshots**
![Screen Shot 2020-09-25 at 3 03 45 PM](https://user-images.githubusercontent.com/11034221/94311253-8d30d900-ff40-11ea-89ec-aaaf26058869.jpg)

**Comment by 123joshuawu** [comment]
This will be userful https://github.com/Akryum/vue-virtual-scroller#dynamicscroller

**Comment by venkatrsrinivas** [comment]
Hello, I was looking into the "Schedule: Visual Padding in Course Elements Issue" and I'm slightly confused on what the Problem exactly is, and if I could get some clarification. I read that you all were talking about using the DynamicScroller and I made a Branch = fixVisualPadCourseElements and starting working on that, but I'm not sure if I'm understanding the Issue itself.

**Comment by jshom** [comment]
Yep, thanks for bringing this up. Issue is a visual issue where a course listing in the search is taking too much room in effect taking away functionality from the schedule page.

Once this issue is done, we should be able to see more courses than we were able to see before.
Since some courses have long names, we can cut off title with an ellipsis if the title takes over two lines.

Let me know if I can answer another question!

**Comment by venkatrsrinivas** [comment]
Alright sounds good. I'll try working on this, and seeing if I can fix the General Padding and be in communication with Josh regarding any Implementation-Specific Details. Thank you both!

**Comment by venkatrsrinivas** [comment]
Hello Everyone. 
I just created a Formal Pull Request for this Issue. 
Hopefully by the end of this week, it is also merged into Master Branch.
Kindly let me know your thoughts and feedback.
Thank you!
